Am adaugat din greseala Locuinta4 in loc de Locuinta_py_json_1.py

Fisierul in care este interactiunea cu json este Locuinta_py_json_1.py !!
